app/assets/javascripts/staff/events.js

```diff
@@ -7,5 +7,9 @@ $(document).ready(function () {
   $('.cancel-status-change').click(function() {
     $('.status-dropdown').hide();
     $('.btn-nav').show();
-  })
-});
\ No newline at end of file
+  });
+
+  $('.modal').on('keypress', function (e) {
+    if (e.which == 13) $(this).submit();
+  });
```

Comment:
Nice. Do you know what was causing the modal to hide on enter before? If there's code that was doing that, I'd want to clean that up in addition to adding this code.